app/presenters/url.rb

```diff
@@ -0,0 +1,22 @@
+class Url
```

Comment:
I thought this class was a substitute for the `URI` class .... which it is not. How about `LinkHelper` or `Link` or something?